funnel/views/proposal.py

```diff
@@ -80,7 +80,7 @@ def proposal_data(proposal):
             ('votes_count', proposal.votes_count()),
             ('votes_groups', proposal.votes_by_group()),
             ('votes_bydate', proposal.votes_by_date()),
-            ('status', proposal.status),
+            ('status', proposal.status.value),
```

Comment:
Use `proposal.status.label`. The value is now private internal detail. (This may break the Android app.)

We should change the `LabeledEnum` to use (value, name, title) tuples and return `proposal.status.label.name` here so that `name` can be a consistent non-localized string. However, all other references to `.label` in the UI will need to become `.label.title`.

Reply:
so it'll be like `SUBMITTED = (1, 'submitted', __("Submitted"))`?